docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,617 @@
+
+.. code-block:: xml
+
+    <depend>rclcpp</depend>
+    <depend>rmw_fastrtps_cpp</depend>
+    <depend>fastrtps</depend>
+    <depend>std_msgs</depend>
+
+
+Configure initial QoS
+---------------------
+
+Ownership Strength Policy is mutable, but Ownership Policy is not. Then, we need to configure EXCLUSIVE_OWNERSHIP_POLICY to all participants before running the ROS nodes.
+To do that, create a new xml file in the root of the workspace:
+
+.. code-block:: bash
+
+    cd ~/ros2_ws
+    touch profiles1.xml
+
+Open the newly created file with your preferred editor and paste the following xml code:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+    <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+        <data_writer profile_name="/chatter">
+            <qos>
+                <ownership>
+                    <kind>EXCLUSIVE</kind>
+                </ownership>
+                <ownershipStrength>
+                    <value>10</value>
+                </ownershipStrength>
+            </qos>
+        </data_writer>
+
+        <data_reader profile_name="/chatter">
+            <qos>
+                <ownership>
+                    <kind>EXCLUSIVE</kind>
+                </ownership>
+            </qos>
+        </data_reader>
+    </profiles>
+
+
+This xml includes one profile for a publisher (data writer) and one profile for a subscriber (data reader), and sets them to exclusive ownership, and ownership strength of value 10 for the publisher.
+This will be applied to the Publisher 1 and to the Subscriber.
+We need another profile in a separate file to assign a different ownership strength to the Publisher 2:
+
+.. code-block:: bash
+
+    touch profiles2.xml
+
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+    <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+        <data_writer profile_name="/chatter">
+            <qos>
+                <ownership>
+                    <kind>EXCLUSIVE</kind>
+                </ownership>
+                <ownershipStrength>
+                    <value>2</value>
+                </ownershipStrength>
+            </qos>
+        </data_writer>
+
+
+This will assign an ownership strength of value 2 to the Publisher 2.
+
+Build
+-----
+
+Now we are ready to build the package.
+Change your directory to the workspace folder and build using colcon:
+
+.. code-block:: bash
+
+    cd ~/ros2_ws
+    colcon build
+
+
+Run
+---
+
+Open three terminals in the workspace folder.
+On each you need to source Vulcanexus installation, as well as the package installation.
+Then, export the `FASTRTPS_DEFAULT_PROFILES_FILE` environment variable to point out to the corresponding profiles file.
+And run the ROS 2 node.
+
+First, in the first terminal, run the subscriber node, configured with the profiles1.xml file:
```

Comment:
I'd recommend using tabs

Reply:
Done

Reply:
The fact is that I would want to still explain things between each code-block, and I couldn't do that using tabs. Do you want me to change the explanation in order to use tabs?